### PR TITLE
Update to the super feature

### PR DESCRIPTION
**Pull request by michaelzap94**
Added much needed text to the REadme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# collaborative-bootcamp
\ No newline at end of file
+# collaborative-bootcamp
+I enjoy JavaScript because it utilizes both Functional programming and Object-oriented programming.
```
Comment (capg84) [inline README.md:2]: Change this please

Comment (michaelzap94) [reply README.md:2]: ok I will

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# collaborative-bootcamp
\ No newline at end of file
+# collaborative-bootcamp
```
Comment (capg84) [inline README.md:1]: Change this , it's broken.

**Review by capg84 — CHANGES_REQUESTED** [review]
Added some comments

File: README.md (hunk 2 above)
Comment (michaelzap94) [reply README.md:1]: ok

**Review by capg84 — APPROVED** [review]
LGTM

**Pull request merged**